src/main/kotlin/org/treeWare/proto3/message/Proto3TypeConverter.kt

```diff
@@ -0,0 +1,47 @@
+package org.treeWare.proto3.message
+
+import org.treeWare.metaModel.FieldType
+import org.treeWare.metaModel.getFieldTypeMeta
+import org.treeWare.metaModel.getMetaName
+import org.treeWare.model.core.EntityModel
+import org.treeWare.model.core.getMetaModelResolved
+
+
+fun getEncodingType(fieldMeta: EntityModel): String =
+    when (val fieldType = getFieldTypeMeta(fieldMeta)) {
+        FieldType.BOOLEAN -> "bool"
+        FieldType.UINT8,
+        FieldType.UINT16,
+        FieldType.UINT32 -> "uint32"
+        FieldType.UINT64 -> "uint64"
+        FieldType.INT8,
+        FieldType.INT16,
+        FieldType.INT32 -> "int32"
+        FieldType.INT64 -> "int64"
+        FieldType.FLOAT -> "float"
+        FieldType.DOUBLE -> "double"
+        FieldType.BIG_INTEGER,
+        FieldType.BIG_DECIMAL,
+        FieldType.STRING,
+        FieldType.UUID,
+        -> "string"
+        FieldType.BLOB -> "bytes"
+        FieldType.TIMESTAMP -> "timestamp"
+        FieldType.PASSWORD1WAY,
+        FieldType.PASSWORD2WAY -> "string"
+        FieldType.ASSOCIATION -> "string"
+        FieldType.ENUMERATION -> snakeToCamel(getEnumerationFieldName(fieldMeta))
+        FieldType.COMPOSITION -> snakeToCamel(getCompositionFieldName(fieldMeta))//throw IllegalStateException("$fieldType encountered")
+        FieldType.ALIAS -> throw IllegalStateException("Unsupported field type: $fieldType")
+        null -> throw java.lang.IllegalStateException("Null field type")
+    }
+
```

Comment:
Don't we already have existing helper functions for the following two functions? If not, we should move these two functions into the appropriate helper function file.